tests/ott/swift/Example/Tests/MultiRequestWithPrimitives.swift

```diff
@@ -73,15 +73,15 @@ class MultiRequestWithPrimitives: QuickSpec {
             done(error)
             
         }
-        requestBuilder.setBody(key: "udid", value: "72958A68-3823-4C67-8A19-ADA920599301")
+        requestBuilder.setParam(key: "udid", value: "72958A68-3823-4C67-8A19-ADA920599301")
```

Comment:
@tan-tan-kanarek, @srivkas: I believe udid value should be taken from tests/ott/swift/Example/Tests/TestConfig.plist. Need to add a placeholder to tests/ott/swift/Example/Tests/TestConfig.template.plist, and also add tests/ott/swift/Example/Tests/TestConfig.template.plist to .gitignore.

Reply:
handled here: https://github.com/kaltura/clients-generator/pull/195